transparency-in-coverage/python/processors/helpers.py

```diff
@@ -9,7 +9,7 @@
 import urllib
 import pathlib
 import logging
-from schema import SCHEMA
+from .schema import SCHEMA
```

Comment:
using local imports has its pros and cons. In this case, this works regardless of where the sources root is.

Generally speaking, it helps when you can run your python projects from the root of the repository folder, or the root of that project folder. Python doesn't force you to do one or the other, so it's makes it easy on the user when you clearly define where you want to run from.